platform-sdk/swirlds-common/src/main/java/com/swirlds/common/wiring/tasks/AbstractTask.java

```diff
@@ -66,8 +67,32 @@ protected final void setRawResult(Void value) {}
      * If the task has no dependencies then execute it. If the task has dependencies, decrement the dependency count and
      * execute it if the resulting number of dependencies is zero.
      */
-    protected void send() {
+    void send() {
         if (dependencyCount == null || dependencyCount.decrementAndGet() == 0) {
+            runTask();
+        }
+    }
+
+    /**
+     * If the task has no dependencies then execute it. If the task has dependencies, decrement the dependency
+     * count by {@code count} and execute if the resulting number of dependencies is zero.
+     *
+     * @param count dependency count to decrement
+     */
+    void send(final int count) {
```

Comment:
I'm against this extension. It's not actually needed (I left another comment in the relevant PR).

Reply:
OK, let's not add it for now.